platform/commonUI/browse/src/creation/CreationPolicy.js

```diff
@@ -0,0 +1,49 @@
+/*****************************************************************************
+ * Open MCT Web, Copyright (c) 2014-2015, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT Web is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT Web includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+/*global define*/
+
+define(
+    [],
+    function () {
+        "use strict";
+
+        /**
+         * A policy for determining whether objects of a certain type can be
+         * created.
+         * @returns {{allow: Function}}
+         * @constructor
+         */
+        function CreationPolicy() {
+            return {
+                /**
+                 * Only allow creation of object types that have the
+                 * Creation capability
+                 */
+                allow: function (type) {
```

Comment:
> For some reason I used example/policy as the exemplar

What a silly thing to do! 

Added a note to #45 ("add narrative to example code") indicating that we need to catch up with platform code style, too.